lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -474,4 +474,14 @@ private function parseColumnCommentFromSQL($column, $sql)
 
         return false;
     }
+    
+    /**
+     * @param string $str
+     * 
+     * @return string
+     */
+    private function pregQuote($str)
+    {
+        return preg_quote($str, '#');
```

Comment:
Why `#`? Patterns are using `/`. This is invalid usage, you are abusing preg_quote for different purpose.

Reply:
Please have a look at the linked issue.

Reply:
I see your point, but it is still invalid usage, `#` is not a delimiter. It should be done i.e. with simple `str_replace('#', '\#', $str)` _after_ passing `$str` through `preg_quote()`.

Reply:
Well I already did that in the old PR, see https://github.com/doctrine/dbal/pull/2884/commits/5bf652ef0ac712116e9f7bbcfc95ba6465c9435c#diff-3ee26e34bba502d9bf80cff65c964b9aR463.